### PR TITLE
feat: update suspension_type mapping

### PR DESCRIPTION
**Pull request by nayib-jose-gloria**
## Reason for Change

- #953

## Changes

- Delete listed, deprecated assay-to-suspension-type mappings
- Add new listed assay-to-suspension-type mappings 
- Bump schema version to 5.2.0 

## Notes for Reviewer

### CODE REVIEW

File: cellxgene_schema_cli/cellxgene_schema/schema_definitions/schema_definition.yaml [hunk 1]
```diff
@@ -397,58 +384,45 @@ components:
             enum:
               - "cell"
               - "nucleus"
-          - # If assay_ontology_term_id is EFO:0009918 or its descendants, 'suspension_type' MUST be 'na'
+          - # If assay_ontology_term_id is EFO:0008994 or its descendants, 'suspension_type' MUST be 'na'
```
Comment (Bento007) [inline cellxgene_schema_cli/cellxgene_schema/schema_definitions/schema_definition.yaml:387]: should this be [EFO:0008992](https://www.ebi.ac.uk/ols4/ontologies/efo/classes?obo_id=EFO%3A0008992)?

File: cellxgene_schema_cli/cellxgene_schema/schema_definitions/schema_definition.yaml [hunk 2]
```diff
@@ -397,58 +384,45 @@ components:
...
-                    - EFO:0008994
+                    - EFO:0002761
                 inclusive: True
             type: categorical
             error_message_suffix: >-
-              when 'assay_ontology_term_id' is EFO:0008994 or its descendants
+              when 'assay_ontology_term_id' is EFO:0002761 or its descendants
             enum:
-              - "na"
-          - # If assay_ontology_term_id is EFO:0008919 or its descendants, 'suspension_type' MUST be 'cell'
```
Comment (Bento007) [inline cellxgene_schema_cli/cellxgene_schema/schema_definitions/schema_definition.yaml:439]: is this an extra change for this PR? I don't see it in the list of ids to delete.

Comment (nayib-jose-gloria) [reply cellxgene_schema_cli/cellxgene_schema/schema_definitions/schema_definition.yaml:439]: it's still [here](https://github.com/chanzuckerberg/single-cell-curation/pull/1007/files#diff-7cd17d19c6ca9ca666aeb04b33d9cc5b5e46100a05bd3076ed3730dfc720af3fR400)!

Not sure why the diff is showing it as deleted, I didn't touch it. Just appended below it

**Review by Bento007 — APPROVED** [review]

File: cellxgene_schema_cli/cellxgene_schema/schema_definitions/schema_definition.yaml (hunk 2 above)
Comment (Bento007) [reply cellxgene_schema_cli/cellxgene_schema/schema_definitions/schema_definition.yaml:439]: I had a feeling that's what it was. Thank you for confirming.

**Pull request merged**